ravenframework/DistributionsND.py

```diff
@@ -0,0 +1,308 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on Nov 22, 2023
+@author: j-bryan
+"""
+
+import numpy as np
+import scipy.stats
+import scipy.linalg
+from .utils import mathUtils
+
+import os,sys
+ravenDir = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(sys.argv[0])),os.pardir,os.pardir,os.pardir,os.pardir))
+frameworkDir = os.path.join(ravenDir, 'framework')
+sys.path.append(ravenDir)
+from ravenframework.utils import utils
+utils.find_crow(frameworkDir)
+from ravenframework.CustomDrivers.DriverUtils import setupCpp
+setupCpp()
+CrowDistribution1D = utils.findCrowModule('distribution1D')
```

Comment:
Is this code temporary? If so can a comment be added explaining when it can be removed?

Reply:
I can remove all of that before this PR is merged. It's just to help get a crow distribution set up to check the SVD results.

Reply:
Okay, sounds good.